src/redux/reducers/authReducer.js

```diff
@@ -0,0 +1,37 @@
+import * as authTypes from '../actionTypes/authTypes'
+
+const initialState = {
+  userInfo: {},
+  accessToken: '',
+  email: '',
+  createdAt: '',
+  isLoading: false,
+  isError: false,
+}
+
+const authReducer = (state = initialState, action) => {
+  const {type, payload} = action
+
+  switch (type) {
+    case authTypes.GET_LOGIN_INFO:
+      return {...state}
+
+    case authTypes.GET_LOGIN_INFO_SUCCESS:
+      return {
+        ...state,
+        userInfo: payload.userInfo,
+        accessToken: payload.userInfo.access_token,
+        email: payload.userInfo.user.email,
+        createdAt: payload.userInfo.user.createdAt,
```

Comment:
we already have userInfo which contains all attributes of user so why need email and createdAt >